### PR TITLE
add github.xverizex.nem-desktop

### PR DESCRIPTION
**Pull request by xverizex**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: github.xverizex.nem-desktop.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "github.xverizex.nem-desktop",
```
Comment (hfiguiere) [inline github.xverizex.nem-desktop.json:2]: this is not a valid app-id.

File: github.xverizex.nem-desktop.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "nem-desktop",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/xverizex/nem-desktop.git",
```
Comment (hfiguiere) [inline github.xverizex.nem-desktop.json:34]: this is repository isn't found. Did you mean `https://github.com/xverizex/nem_desktop.git` ?

albeit it doesn't have a tag.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: github.xverizex.nem-desktop.json (hunk 2 above)
Comment (xverizex) [reply github.xverizex.nem-desktop.json:34]: I create a tag via the website. Help me with a valid app-id.

Comment (xverizex) [reply github.xverizex.nem-desktop.json:34]: https://github.com/xverizex/nem_desktop is valid

**Comment by xverizex** [comment]
I use --permission=home:create for storage crypto keys and audio files.
I use --permission=pulseaudio for short sound notifications.

**Comment by xverizex** [comment]
Now is ok?

File: github.xverizex.nem-desktop.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "github.xverizex.nem_desktop",
```
Comment (hfiguiere) [inline github.xverizex.nem-desktop.json:2]:
```suggestion
    "app-id" : "io.github.xverizex.nem_desktop",
```

Comment (hfiguiere) [reply github.xverizex.nem-desktop.json:2]: that also mean renaming the manifest file.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: github.xverizex.nem-desktop.json (hunk 3 above)
Comment (xverizex) [reply github.xverizex.nem-desktop.json:2]: renaming with io.?

Comment (xverizex) [reply github.xverizex.nem-desktop.json:2]: Do I need to do this myself?

Comment (xverizex) [reply github.xverizex.nem-desktop.json]: got it. I made.

File: io.github.xverizex.nem_desktop [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "github.xverizex.nem_desktop",
```
Comment (hfiguiere) [inline io.github.xverizex.nem_desktop:2]:
```suggestion
    "app-id" : "io.github.xverizex.nem_desktop",
```

This wasn't done.

Comment (hfiguiere) [reply io.github.xverizex.nem_desktop:2]: and the manifest name is mssing the extension `.json`

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
You never fixed the app-id as requested. And that make the manifest filename still wrong as well (it was just missing the extension last time).

**Comment by xverizex** [comment]
did

**Comment by hfiguiere** [comment]
Filename still wrong.

**Comment by xverizex** [comment]
rename to io.github.xverizex.nem_desktop.json?

**Comment by xverizex** [comment]
I will learn.

**Comment by hfiguiere** [comment]
bot, build io.github.xverizex.nem_desktop

File: io.github.xverizex.nem_desktop.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    "modules" : [
+        {
+            "name" : "nem_desktop",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/xverizex/nem_desktop.git",
+                    "tag" : "0.1.2",
```
Comment (hfiguiere) [inline io.github.xverizex.nem_desktop.json:35]: this tag doesn't exist.

```suggestion
                    "tag" : "v0.1.2",
```

**Comment by xverizex** [comment]
ok

**Comment by xverizex** [comment]
I changed the domain.

**Comment by hfiguiere** [comment]
bot, build io.github.xverizex.nem_desktop

**Comment by hfiguiere** [comment]
```
io.github.xverizex.nem_desktop.desktop: AppData problem: tag-missing : <description> required
```
the appdata file is missing the `<description>` tag.

**Comment by xverizex** [comment]
I did a description.

**Comment by xverizex** [comment]
please, run the bot.

**Comment by vchernin** [comment]
You can run the bot by just commenting the command :)
bot, build io.github.xverizex.nem_desktop

**Comment by vchernin** [comment]
`io.github.xverizex.nem_desktop.desktop: AppData problem: tag-missing : <description> required`
Tag is still missing

BTW you can verify an appdata file locally without buildbot or flatpak-builder. [This should work](https://github.com/flathub/flathub/wiki/App-Requirements):
```
flatpak install org.freedesktop.appstream-glib
flatpak run org.freedesktop.appstream-glib validate <appdata-file>
```

**Comment by xverizex** [comment]
bot, build io.github.xverizex.nem_desktop

**Comment by xverizex** [comment]
wow.

**Comment by xverizex** [comment]
bot, build io.github.xverizex.nem_desktop

**Comment by xverizex** [comment]
are now closing a pull request?

**Comment by xverizex** [comment]
bot, build io.github.xverizex.nem_desktop

**Comment by xverizex** [comment]
whats is next?

**Comment by hfiguiere** [comment]
patience. one of the admin will either request change or approve it.

**Comment by xverizex** [comment]
bot, build io.github.xverizex.nem_desktop

**Pull request closed without merging**